vars/withAksClient.groovy

```diff
@@ -18,10 +16,14 @@ def withRegistrySecrets(Closure block) {
 }
 
 
-def call(String subscription, Closure block) {
+def call(String subscription, String environment, Closure block) {
   withDocker('hmcts/cnp-aks-client:az-2.0.50-kubectl-1.12.2-helm-2.11', null) {
     withSubscription(subscription) {
       withRegistrySecrets {
+        def envName = environment.toUpperCase()
+        echo "envname : ${envName} "
+        env.AKS_CLUSTER_NAME = env."${envName}_AKS_CLUSTER_NAME" ?: "cnp-${environment}-cluster"
```

Comment:
do we need the fallbacks here?
can't we just rely on the env variables always being set?

Reply:
Ideally if we had good naming convention, we wouldn't need have to set them in environment. I would rather avoid the need of setting them  by following a convention.